recipes/postgrest/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "postgrest" %}
```

Comment:
We should mirror the PyPI name here, too.

Reply:
postgrest is the correct name, postgrest pypi package is dead and conda convention is to omit "-python" when this happens (e.g. https://github.com/conda-forge/perspective-feedstock is "perspective-python" on pypi, but perspective on conda due to stale package)